.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
```

Comment:
We also have a process for new audits we can reference https://github.com/GoogleChrome/lighthouse/blob/master/docs/new-audits.md

Reply:
went ahead and added this to the feature request template